truss/templates/server/model_wrapper.py

```diff
@@ -596,7 +600,7 @@ async def preprocess(
         )
         return await self._execute_async_model_fn(inputs, request, descriptor)
 
-    async def predict(
+    async def _predict(
```

Comment:
Previously `__call__` was the orchestrator, and this function just wrapped the invocation of `predict`. Now I've made `predict` the entrypoint + orchestrator, and this private function is just the singular `predict` wrapper